server/src/Persistence.hs

```diff
@@ -155,7 +156,12 @@ loadFromBackend backend config = runExceptT $ do
   return val
 
 syncToBackend :: StorageBackend -> PersistentValue -> IO ()
-syncToBackend File pv = syncFile pv
+syncToBackend File pv = do
+    start <- Clock.getTime Clock.Monotonic
+    syncFile pv
+    end <- Clock.getTime Clock.Monotonic
+    let time = Clock.toNanoSecs $ (Clock.diffTimeSpec end start) `div` 1000000
```

Comment:
In the end, it probably doesn't matter, but isn't the order of operations conceptually wrong here?

The `Clock.toNanoSecs` spans the whole division, so despite its name, it returns milliseconds.

Just removing the $ should result in the same output in a less confusing way:  ``Clock.toNanoSecs (Clock.diffTimeSpec end start) `div` 1000000``

Reply:
Yep, it's conceptually better without the `$`. I'll remove that!